doc/CONTRIBUTING.md

```diff
@@ -40,6 +40,21 @@ The mainline branch is `develop` (this will change post-1.0.0). Feature developm
 Name your feature branches with a name or handle of the branch owner as a prefix, followed by a brief description preceded by a slash, e.g. `cwgoes/eal-inference-bugfix`.
 
 Pushing minor changes (typo fixes) to another person's branch is fine. Ask before pushing major changes.
+## Version changelogs
+
+The format for the changelog follows the PVP format below.
+
+V A.B.C.D
+
+A and B are both major versions. C is the Minor version, and D is the patch, in other words it looks like
+
+V Major.Major.Minor.Patch
+
+Major release will bump the major version, milestones bump the minor
+version, and pull requests bump the patch number.
```

Comment:
Do you have a reference about this? I don't think pull requests have anything to do with patch, minor or major version. E.g. if a PR brings breaking changes, I think it should update the changelog file with a major bump

Reply:
Fair, I think I worded it poorly, I guess I should specify "a pull request should bump one of these numbers)